server/channels/store/storetest/session_store.go

```diff
@@ -397,6 +398,71 @@ func testGetSessionsExpired(t *testing.T, rctx request.CTX, ss store.Store) {
 	}
 }
 
+func testGetSessionsWithActiveDeviceIds(t *testing.T, rctx request.CTX, ss store.Store) {
```

Comment:
This appears to be a more comprehensive version of an earlier test, so I've just opted to keep both.